SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -21,6 +21,7 @@ USpatialGDKSettings::USpatialGDKSettings(const FObjectInitializer& ObjectInitial
 	, EntityCreationRateLimit(0)
 	, OpsUpdateRate(1000.0f)
 	, bEnableHandover(true)
+	, MaxNetCullDistanceSquared(900000000.0f) // Set to twice the default Actor NetCullDistanceSquared (300m)
```

Comment:
Should we provide an easy way of disabling this clamping besides setting this to some huge value? E.g. setting this to 0?

Reply:
On a second thought, probably not worth additional effort considering this is temporary until the runtime issue is fixed.

Reply:
Good idea

Reply:
Oh - only saw the first comment 🤕 
Done it anyway.